src/h5web/providers/mock/MockProvider.tsx

```diff
@@ -9,13 +10,18 @@ interface Props {
 
 function MockProvider(props: Props): JSX.Element {
   const { domain, children } = props;
-  const [api, setApi] = useState<MockApi>();
 
-  useEffect(() => {
-    setApi(new MockApi(domain));
-  }, [domain]);
-
-  return <Provider api={api}>{children}</Provider>;
+  return (
+    <Provider
+      api={{
+        domain,
+        getMetadata: async () => mockMetadata,
+        getValue: async (id: HDF5Id) => (mockValues as HDF5Values)[id],
```

Comment:
The API is now so simple that I can just inline it here! Note that I cast `mockValues` here instead of typing `mockValues` in `data.ts`, so that I can just write `mockValues.twoD` in the stories and get a `number[][]`.